client/src/app/Icons/Icons.js

```diff
@@ -6,6 +6,7 @@ import { ReactComponent as SkipIcon } from './assets/skip.svg'
 import { ReactComponent as CrossIcon } from './assets/x.svg'
 import { ReactComponent as ResetIcon } from './assets/reset.svg'
 import { ReactComponent as BinIcon } from './assets/bin.svg'
+import { ReactComponent as ArrowIcon } from './assets/arrow.svg'
```

Comment:
You could use the "organize imports" command in VS Code in order to have them all sorted the same way. :)